### ISSUE TITLE
Missing TLS certificate verification

### ISSUE THREAD

**Issue by mcatanzaro** [intel/dleyna-server#169]
Hi, libdylena's device.c does not appear to be enabling TLS certificate verification on the SoupSessionAsync object it creates. [You have to do this manually when using the deprecated SoupSessionAsync.](https://blogs.gnome.org/mcatanzaro/2021/05/25/reminder-soupsessionsync-and-soupsessionasync-default-to-no-tls-certificate-verification/)

**Comment by phako** [comment]
I suppose the linked fix above should suffice?

**Comment by mcatanzaro** [comment]
> I suppose the linked fix above should suffice?

That's the best way to fix this issue, yes, [assuming you've read the porting guide](https://developer-old.gnome.org/libsoup/stable/libsoup-session-porting.html) and are OK with the differences.

**Comment by mcatanzaro** [comment]
Note there is an identical issue in dleyna-renderer: https://github.com/intel/dleyna-renderer/issues/171

**Comment by phako** [comment]
Yes that should not matter here. I have fixed the renderer issue as well, the commit should be referenced in that ticket.
